pythonforandroid/bdistapk.py

```diff
@@ -143,6 +143,14 @@ class BdistAAR(Bdist):
     package_type = 'aar'
 
 
+class BdistAAB(Bdist):
+    """
+    distutil command handler for 'aab'
+    """
```

Comment:
Nitpick: oneline docstring should fit in one line https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

Reply:
Fixed, this one and the ones immediately before.